src/ctia/entity/identity.clj

```diff
@@ -85,6 +86,9 @@
      :groups em/token}}})
 
 (defrecord IdentityStore [state]
+  IStore
+  (close [this]
+    (close-cm! (:state this)))
```

Comment:
I do not know why yet but IdentityStore is has the schema:
``` clojure
{:state StoreSchema}
```
while the other stores have the schema StoreSchema